### PR TITLE
Update Programming.md

### PR DESCRIPTION
**Pull request by StoyanDimitrov**
Cosmetic changes and minor formatting adjustments

### CODE REVIEW

File: Readme_files/Programming.md [hunk 1]
```diff
@@ -1,99 +1,109 @@
...
+In any case, the firmware will check the settings and reset them if not valid.
 
 ### STM32 not recognized error
-Some STM32 have the debug port disabled, in this case the only way to access is connecting nRST pin to the ST-Link.<br>
-There're several ST-Link clones with a flaw in nRST pin, which is wired wrong by design, and won't able to connect.<br> 
-Temporaly short nRST (STM32 pin 7) to gnd, click "connect" button, wait 1-2 seconds and release nRST, now it should recognice it.<br>
-Usually nRST is connected to a capacitor, only requiring to short the capacitor as showed in this picture:<br>
-<img src="/Readme_files/stlink_force_rst.jpg?raw=true"><br><br>
+Some STM32 have their debug port disabled, in this case the only way to access is connecting `nRST` pin to the ST-Link.<br>
+There're several STM32 clones with a flaw in `nRST` pin, which is wired wrong by design, and won't able to connect.<br>
```
Comment (StoyanDimitrov) [inline Readme_files/Programming.md]: The original text is "ST-Link clones" I suppose the intend was to be "STM32 clones". Am I right?

Comment (deividAlfa) [reply Readme_files/Programming.md]: Nope, it really means ST-Link clones from AliExpress and such, some had wrong pcb layout, the pin that toggled nRST wasn't connected to the programming connector, so it didn't work.

Comment (deividAlfa) [reply Readme_files/Programming.md]: Here you can see what I mean:
https://ziutek.github.io/2018/03/26/stlink.html

Comment (StoyanDimitrov) [reply Readme_files/Programming.md]: I fixed that.

**Pull request merged**